Sources/KeystoreKey.swift

```diff
@@ -136,6 +136,23 @@ public struct KeystoreKey {
         let key = try decrypt(password: password)
         return try Secp256k1.shared.sign(hash: hash, privateKey: key)
     }
+
+    /// Signs multiple hashes with the given password.
+    ///
+    /// - Parameters:
+    ///   - hashes: array of hashes to sign
+    ///   - password: key password
+    /// - Returns: [signature]
+    /// - Throws: `DecryptError` or `Secp256k1Error`
+    public func signHashes(hashes: [Data], password: String) throws -> [Data] {
```

Comment:
Let's make this function `public func signHashes(_ hashes: [Data], password: String) throws -> [Data]` (notice the underscore) to avoid having the word hashes twice when calling.